soco/data_structures.py

```diff
@@ -1012,7 +1012,18 @@ class DidlMusicGenre(DidlGenre):
 
 class ListOfMusicInfoItems(list):
 
-    """Abstract container class for a list of music information items."""
+    """Abstract container class for a list of music information items.
+
+    Instances of this class are returned from queries into the music library
+    or to music services. The attributes :attr:`~total_matches` and
+    :attr:`~number_returned` are used to ascertain whether paging is required
+    in order to retrive all elements of the query. :attr:`~total_matches` is
+    the total number of results to the query and :attr:`~number_returned` is
+    the number of results actually returned. If the two differ, paging is
+    required. Paging is typically performed with the ``start`` and
+    ``max_items`` arguments to the query method.
```

Comment:
Just one thing: A link to `music_library.get_music_library_information` would be good.